internal/connector/internal/handlers/annotations.go

```diff
@@ -0,0 +1,68 @@
+package handlers
```

Comment:
Moved from the `connectors.go` file into this one as the functions are shared between `connector`'s and `processor`'s.